src/App.js

```diff
@@ -1,39 +1,46 @@
 import React, {Component} from 'react'
-import './App.css';
-import {movieData} from './movieData'
+import './App.css'
 import Movies from './Movies'
 import MovieDetails from './MovieDetails'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: movieData.movies,
-      singleMoviePreview: false
+      movies: [],
+      singleMoviePreview: false,
+      id: 0
     }
   }
 
-  handleMovieClick = () => {
-    // console.log('hey im a movie click', this.state.singleMoviePreview)
-    this.setState({singleMoviePreview: true})
-    // console.log(this.state, 'after')
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+      .then(response => response.json())
+      .then(data => this.setState({ movies: data.movies }))
+      .catch(error => alert(`Error: ${error}`))
   }
 
-  // handleClose() {
-  //
-  // }
+  handleMovieClick = (id) => {
+    this.setState({singleMoviePreview: true, id: id})
+  }
 
+  handleClose = () => {
+    this.setState({singleMoviePreview: false, id: 0})
+  }
 
   render() {
     return (
       <body>
         <header>
           <h1>Rancid Tomatillos</h1>
         </header>
-        {this.state.singleMoviePreview && <MovieDetails />}
-        <main>
-          <Movies movies={this.state.movies} handleMovieClick={this.handleMovieClick}/>
-        </main>
+        {this.state.singleMoviePreview ? 
```

Comment:
Good conditional logic here.  This will likely get refactored now that you have React Router, but I like how you handled this!